sdks/python/tox.ini

```diff
@@ -25,6 +25,7 @@ select = E3
 
 [testenv:py27]
 deps=
+  nose
```

Comment:
New. Running autocomplete_test outside the testing framework depends on this. (I think the other option is to add it as an install_requires package since tox does setup.py install at first. Adding it as tests_require was not enough.)